#
# Submitted-By: Robert Schwebel, 2006-05-25
# Committed-By: Robert Schwebel
#
# Error:
#
# sulogin.c: In function 'main':
# sulogin.c:444: warning: implicit declaration of function 'getpgid'
# sulogin.c:444: warning: nested extern declaration of 'getpgid'
# sulogin.c:450: warning: implicit declaration of function 'getsid'
# sulogin.c:450: warning: nested extern declaration of 'getsid'
#
# Description:
#
# getpgid() is only defined in unistd.h if __USE_XOPEN_EXTENDED is set.
#
# State:
# 
# unknown
#

diff -urN initng-0.6.0/tools/sulogin.c initng-0.6.0-ptx/tools/sulogin.c
--- initng-0.6.0/tools/sulogin.c	2006-03-27 11:02:18.000000000 +0200
+++ initng-0.6.0-ptx/tools/sulogin.c	2006-05-25 13:55:34.000000000 +0200
@@ -17,6 +17,8 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+/* define this to get getpgid() */
+#define __USE_XOPEN_EXTENDED
 #include <unistd.h>
 #include <fcntl.h>
 #include <signal.h>
